src/mame/sinclair/atm.h

```diff
@@ -0,0 +1,106 @@
+// license:BSD-3-Clause
+#ifndef MAME_INCLUDES_ATM_H
+#define MAME_INCLUDES_ATM_H
+
+#pragma once
+
+#include "emu.h"
```

Comment:
You can’t include emu.h from headers.  It has to be the the first thing included from a .cpp file.  If this isn’t done, it causes issues with inconsistent behaviour with prefix headers on/off, and can also cause issues with Visual Studio build.